src/home_robot_hw/home_robot_hw/env/stretch_pick_and_place_env.py

```diff
@@ -28,11 +29,93 @@
 from home_robot_hw.remote import StretchClient
 from home_robot_hw.utils.grasping import GraspPlanner
 
+RECEP_START_IDX = 4
```

Comment:
We were changing these values during our debugging session today. Can you make it easier to use the config so that we dont need to change these values

Reply:
Removed this now.